src/redux/state.js

```diff
@@ -81,18 +101,71 @@ let store = {
       this._state.MESSAGES.POST_DATA.push(newPost);
       this._state.MESSAGES.NEW_POST_TEXT = "";
       this._callSubscriber(this._state);
-    } else {
+    }
+    break;
+  case action.type === UPDATE_NEW_MESSAGE_TEXT:
+    {
+      this._state.DIALOGS.NEW_MESSAGE_TEXT = action.body;
+      this._callSubscriber(this._state);
+    }
+    break;
+  case action.type === SEND_MESSAGE:
+    {
+      let body = this._state.DIALOGS.NEW_MESSAGE_TEXT;
+      this._state.DIALOGS.NEW_MESSAGE_TEXT = "";
+      this._state.DIALOGS.CONVERSATION_MESSAGES.push({ text: body, id: 7 });
+      this._callSubscriber(this._state);
+    }
+    break;
+  default:
+    {
       this._state.MESSAGES.NEW_POST_TEXT = action.newPostText;
       this._callSubscriber(this._state);
     }
```

Comment:
The switch/case should be rewritten. Please check the documentation on how to use switch/case: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/switch